certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Gratavid",
+    "description": "Gratavid is a SaaS service that allows organizations to collect and send personalized videos.",
+    "version": "1.0",
+    "contact": {
+      "name": "Gratavid Support",
+      "url": "https://gratavid.zendesk.com/",
+      "email": "contact@gratavid.com"
+    }
+  },
+  "host": "api.gratavid.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/manageIntegrations": {
+      "x-ms-notification-content": {
+        "description": "Id Response",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "noteTitle": {
+              "type": "string",
+              "description": "noteTitle",
+              "title": "Gratavid Note Title"
+            },
+            "noteURL": {
+              "type": "string",
+              "description": "noteURL",
+              "title": "Gratavid Note URL"
+            },
+            "contact_customUserId": {
+              "type": "string",
+              "description": "Gratavid Custom UserId of the contact",
+              "title": "Contact Gratavid Custom User Id"
+            },
+            "contact_customAccountId": {
+              "type": "string",
+              "description": "Gratavid Custom AccountId of the contact",
+              "title": "Contact Gratavid Custom Account Id"
+            },
+            "contact_email": {
+              "type": "string",
+              "description": "contact_email",
+              "title": "Contact Email"
+            },
+            "contact_firstName": {
+              "type": "string",
+              "description": "contact_firstName",
+              "title": "Contact First Name"
+            },
+            "contact_lastName": {
+              "type": "string",
+              "description": "contact_lastName",
+              "title": "Contact Last Name"
+            },
+            "replyText": {
+              "type": "string",
+              "description": "replyText",
+              "title": "Reply Message"
+            },
+            "replyVideoSrc": {
+              "type": "string",
+              "description": "replyVideoSrc",
```

Comment:
@Wtrapp The description values of properties should be free from grammatical and spelling errors as they will be visible to public. 

Docs - [Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@Wtrapp Applies to all the description values

Reply:
Fixed in the latest commit.